### ISSUE TITLE
Ability to create a note per document

### ISSUE THREAD

**Issue by klausondrag** [burtonator/polar-bookshelf#682]
**Is your feature request related to a problem? Please describe.**
I want to create a note for every documents I read. Specifically, I want to create short summaries for with advantages and disadvantages. This fits the scope of polar because it helps with building knowledge and progressive reading.

**Describe the solution you'd like**
A textbox in the annotations side-bar, on top of all the other annotations. It should behave similar to a comment of an annotation except it should be easily editable.

**Describe alternatives you've considered**
- Keeping notes in Zotero: the workflow is not good. Too many clicks, to cumbersome.
- Keeping notes in an editor / separate file: hard to keep the two in sync because the text/markdown file has to refer unambiguously to the pdf/website.
- Also, both alternatives are solutions which separate reading from creating summaries. In the spirit of polar, it should be progressive. While reading the article, while scrolling up and down. 
- Creating many notes per document instead of a single one is a possibility but I would not need that. A summary with advantages and disadvantages can still fit a single note with line breaks. 
- The label "Notes" might be misleading because of the plural.

**Additional context**
Note that this is distinct from https://github.com/burtonator/polar-bookshelf/issues/667 because the notes are still tied to a single document.

Current view:
![polar-feature-notes-1](https://user-images.githubusercontent.com/18405064/53772610-e8bef000-3ede-11e9-9684-d584828bcf52.png)

Suggested view:
![polar-feature-notes-2](https://user-images.githubusercontent.com/18405064/53772611-e9578680-3ede-11e9-95f4-fbefbd8e1db3.png)

**Comment by burtonator** [comment]
Could we have this as sort of a 'user summary' of the document?

I don't know what we would call it just yet.  I want to reserve the field 'summary' for the data about the document which is more objective and possibly provided by the author.  

maybe summarization?? This would be YOUR version of the summary.

Come to think of it , it might be nice to do this with titles too. Like there is the author's title and then you could have your OWN title..  

Maybe I have to keep this metadata in two separate objects.

**Comment by klausondrag** [comment]
You can title it related to summary but at least for me I want to add personal remarks, highlight advantages and disadvantages of papers for example. Or note a similarity to another article. If you strictly want to keep it as a summary in one sentence for example then I suggest introducing another field for general notes.

**Comment by burtonator** [comment]
Yeah.. that's exactly what I meant.  Though maybe it actually should be its own object in and of itself. 

Is this LITERALLY a summary of the document?  Like you're summarizing it?  Or is it more of just like a 'key points' note?

We can support both use cases I'm just thinking maybe it might nice to have some specific fields.

**Comment by sotte** [comment]
I would also like a **general text field for notes** and I can put there whatever I want. No need to specify that it's a summary or key points I would say!

**Comment by ankostis** [comment]
(hope i'm not digressing)
This feature request has a lot in common with what W3C's _web-annotation_ standard envisioned,
and attempted by [Hypothes.is](https://web.hypothes.is/) to implement.
